### DISCUSSION TITLE
EDIT DEFAULT CAPROVER PAGE

### DISCUSSION THREAD

**Discussion by toure5013** [caprover/caprover#1125] [Q&A]
![image](https://user-images.githubusercontent.com/44227093/121812976-993d0200-cc59-11eb-95bb-36f5e09727b5.png)
Hello everyone,

Is anyone have ideas about how to edit caprover default(fallback) page or make redirection ?
Thanks in advance.

**Comment by githubsaturn** [comment]
It's similar to this: https://github.com/caprover/caprover/issues/1025
